Mathlib/Algebra/Star/Pi.lean

```diff
@@ -4,13 +4,14 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Eric Wieser
 -/
 import Mathlib.Algebra.Star.Basic
+import Mathlib.Algebra.Notation.Pi.Defs
 import Mathlib.Algebra.Ring.Pi
 
 /-!
-# `star` on pi types
+# Basic Results about Star on Pi Types
 
-We put a `Star` structure on pi types that operates elementwise, such that it describes the
-complex conjugation of vectors.
+This file provides basic results about the star on product types defined in
+`Mathlib.Algebra.Notation.Pi.Defs`.
```

Comment:
Links to files should be formatted as
```suggestion
`Mathlib/Algebra/Notation/Pi/Defs.lean`.
```

Reply:
Thanks! I was wondering about that. I should have looked at another file to compare.